lib/scholar/linear/polynomial_regression.ex

```diff
@@ -0,0 +1,190 @@
+defmodule Scholar.Linear.PolynomialRegression do
```

Comment:
I think this might not be the correct localization of this module.

Reply:
Where do you think it makes the most sense to have this module?

Reply:
Given we have Scholar.Preprocessing and that sklearn has this under preprocessing, perhaps Scholar.Preprocessing.PolynomialRegression would fit?

However, I'm ok with keeping this here. Why do you think it should be elsewhere, @msluszniak ?

Reply:
I think that if this module has `fit` and `predict` functions and it forms a struct similar to that of LinearRegression, it makes sense to have the module next to LinearRegression

Reply:
:+1: for `Scholar.Preprocessing.PolynomialRegression`.

Reply:
Actually, @polvalente convinced me to keep it in Linear. :)

Reply:
Yeah, I also though about it and now i'm also in favour to keep it in Linear